runtime/nls/cfre/cfrerr.nls

```diff
@@ -1512,3 +1512,11 @@ J9NLS_CFR_ERR_RECORD_COMPONENT_DESCRIPTOR_NOT_UTF8.explanation=Please consult th
 J9NLS_CFR_ERR_RECORD_COMPONENT_DESCRIPTOR_NOT_UTF8.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_RECORD_COMPONENT_DESCRIPTOR_NOT_UTF8.user_response=Contact the provider of the classfile for a corrected version.
 # END NON-TRANSLATABLE
+
+# RecordComponent is not translatable
+J9NLS_CFR_RECORD_CLASS_MUST_BE_FINAL=RecordComponent is implicitly final
```

Comment:
Is `RecordComponent` the right word here?  The check is that the Record class itself is final (and can't be subclasses).  I would expecte `RecordComponent` to correspond to the fields in the Record.  Is that incorrect?

Reply:
woops you're right. will update

Reply:
done.